GPT-87 New styling for the Geoscience Portal website.

heyagosper opened the pull request:
Groovy stuff in this commit (as in cool and hoopy, not the Groovy programming language):
- first cut of redesign, most functionality working: Simple search, advanced search link, clear search, latlong indicator, some links.

heyagosper committed:
GPT-87 New styling for the Geoscience Portal website.
Groovy stuff in this commit:
* first cut of redesign, most functionality working: Simple search, advanced search link, clear search, latlong indicator, some links.

oehm-smith commented:
Looks good to me.

heyagosper committed:
GPT-87 minifier not happy with training comma:-(

oehm-smith commented:
Check this out http://chris.beams.io/posts/git-commit/ (how to write commit messages) :)

heyagosper committed:
GPT-87 Removed call to AppEvents.broadcast because it was recursive

This call prevented the map reset handler loop from working. The first layer was removed but then the loop exited.
